### PR TITLE
Change base image from 17.05.0-ce-dind to stable-dind

### PR DESCRIPTION
**Pull request by delfer**
17.05.0 is not stable version and latest stable version 17.09 have some significant changes.

### CODE REVIEW

**Comment by tboerger** [comment]
I'm fine to update the base to 17.09, but I don't like the switch to the stable tag since the build isn't reproducible anymore if there comes another stable release.

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # docker build --rm -f docker/Dockerfile -t plugins/docker .
 
-FROM docker:17.05.0-ce-dind
+FROM docker:stable-dind
```
Comment (bradrydzewski) [inline docker/Dockerfile]: Please use a fixed version number `17.10.0-ce-dind`

**Comment by delfer** [comment]
@bradrydzewski May be better use `docker@sha256:eaf9c833c2f9fc049fa443f8739986898520228550a90e3f008ba6c7604ce131` for reproducibility?

**Comment by delfer** [comment]
@bradrydzewski I've rewited commit

**Comment by delfer** [comment]
@bradrydzewski changes made, please approve

**Comment by delfer** [comment]
@tboerger how to remind @bradrydzewski to accept PR?

**Comment by delfer** [comment]
At hub.docker.com it's still version from 2017-11-09

**Comment by tboerger** [comment]
There was an issue with the webhook, should be solved now.

**Comment by delfer** [comment]
@tboerger yes, thank you!

**Pull request merged**